apis/fluent-en_GB/atrium-api-fluent-en_GB-common/src/main/kotlin/ch/tutteli/atrium/api/fluent/en_GB/throwableAssertions.kt

```diff
@@ -38,3 +39,36 @@ fun <T : Throwable> Expect<T>.message(assertionCreator: Expect<String>.() -> Uni
  */
 fun <T : Throwable> Expect<T>.messageContains(expected: Any, vararg otherExpected: Any): Expect<T> =
     message { contains(expected, *otherExpected) }
+
+
+/**
+ * Expects that the property [Throwable.cause] of the subject *is a* [TSub] (the same type or a sub-type),
+ * creates an [Expect] of the [TSub] type for it and returns it.
+ *
+ * @return The newly created [Expect] for the property [Throwable.cause] of the subject of the assertion
+ * @throws AssertionError Might throw an [AssertionError] if the assertion made is not correct.
+ *
+ * @since 0.10.0
+ */
+@Suppress("RemoveExplicitTypeArguments")
+inline fun <reified TSub : Throwable> Expect<out Throwable>.cause(): Expect<TSub> =
+    ExpectImpl.throwable.cause(this, TSub::class).getExpectOfFeature()
+
+/**
+ *
+ * Expects that the property [Throwable.cause] of the subject *is a* [TSub] (the same type or a sub-type) and
+ * holds all assertions the given [assertionCreator] creates for it and returns this assertion container.
+ *
+ * Notice, in contrast to other assertion functions which expect an [assertionCreator], this function returns not
+ * [Expect] of the initial type, which was some type `T `, but an [Expect] of the specified type [TSub].
+ *
+ * @return This assertion container to support a fluent API.
```

Comment:
Please use the new phrasing
```suggestion
  * @return An [Expect] for the current subject of the assertion.
```